cfme/containers/provider/__init__.py

```diff
@@ -576,34 +576,32 @@ def set_label(self, name, value):
                 .format(name, value, self.__class__.__name__, self.name, results))
         return results
 
-    def remove_label(self, name, silent_failure=False):
+    def remove_label(self, object, name, value):
         """Remove label by name.
+        :var: object: the object from which the label is being removed.
         :var: name: name of label
-        :var: silent_failure: whether to raise an error or not in case of failure.
-
+        :var: value: the value of the label is being removed.
         Returns:
             :py:type:`bool` pass or fail
         Raises:
             :py:class:`LabelNotFoundException`.
         """
-        json_content = self._get_json()
-        if name not in json_content['metadata'].get('labels', {}).keys():
-            failure_signature = 'Could not find label "{}", labels: {}' \
-                .format(name, json_content['metadata']['labels'])
-            if silent_failure:
-                logger.warning(failure_signature)
-                return False
-            else:
-                raise exceptions.LabelNotFoundException(failure_signature)
-        self.provider.cli.run_command(
-            'oc label {} {} {}-'.format(
-                self._cli_resource_type,
-                ('sha256:{}'.format(self.sha256)
-                 if (self.__class__.__name__ == 'Image') else self.name),
-                name
-            )
+        if hasattr(self, 'project_name'):
+            results = self.provider.cli.run_command('oc project {}'.format(self.project_name))
+            assert results.success, 'Could not set project {}. SSH Results: {}'.format(
+                self.project_name, results)
+        if self.__class__.__name__ == 'Replicator':
```

Comment:
Please ignore the changes on __init__.py at the moment. @gshefer 's [PR](https://github.com/ManageIQ/wrapanapi/pull/161) should offer a better solution. This is a temp measure for the test to run successfully